README.md

```diff
@@ -1,13 +1,53 @@
-
-**Official repository for core projects comprising the Celo platform**
+<p align="center">
+  <a href="https://celo.org/">
+    <img src="https://i.imgur.com/fyrJi0R.png" alt="celo logo" title="Go to celo.org" width="600" style="border:none;"/>
+  </a>
+</p>
+
+<!-- row 1 - status -->
+[![CircleCI](https://img.shields.io/circleci/build/github/celo-org/celo-monorepo/master)](https://circleci.com/gh/celo-org/celo-monorepo/tree/master)
+[![Codecov](https://img.shields.io/codecov/c/github/celo-org/celo-monorepo)](https://codecov.io/gh/celo-org/celo-monorepo)
+[![GitHub contributors](https://img.shields.io/github/contributors/celo-org/celo-monorepo)](https://github.com/celo-org/celo-monorepo/graphs/contributors)
+[![GitHub commit activity](https://img.shields.io/github/commit-activity/w/celo-org/celo-monorepo)](https://github.com/celo-org/celo-monorepo/graphs/contributors)
+[![GitHub Stars](https://img.shields.io/github/stars/celo-org/celo-monorepo.svg)](https://github.com/celo-org/celo-monorepo/stargazers)
+![GitHub repo size](https://img.shields.io/github/repo-size/celo-org/celo-monorepo)
+[![GitHub](https://img.shields.io/github/license/celo-org/celo-monorepo?color=blue)](https://github.com/celo-org/celo-monorepo/blob/master/LICENSE)
+
+<!-- row 2 - links & profiles -->
+[![Website celo.org](https://img.shields.io/website-up-down-green-red/https/celo.org.svg)](https://celo.org)
+[![Blog](https://img.shields.io/badge/blog-up-green)](https://medium.com/celohq)
+[![docs](https://img.shields.io/badge/docs-up-green)](https://docs.celo.org/)
+[![Youtube](https://img.shields.io/badge/YouTube%20channel-up-green)](https://www.youtube.com/channel/UCCZgos_YAJSXm5QX5D5Wkcw/videos?view=0&sort=p&flow=grid)
+[![forum](https://img.shields.io/badge/forum-up-green)](https://forum.celo.org)
+[![Discord](https://img.shields.io/badge/Discord-29%20online-brightgreen)](https://discord.gg/JvJ66Wc)
+[![Twitter Follow](https://img.shields.io/twitter/follow/celohq?style=social)](https://twitter.com/celohq)
+[![Subreddit subscribers](https://img.shields.io/reddit/subreddit-subscribers/CeloHQ?style=social)](https://www.reddit.com/r/CeloHQ/)
+
+<!-- row 3 - detailed status -->
+[![GitHub pull requests by-label](https://img.shields.io/github/issues-pr-raw/celo-org/celo-monorepo)](https://github.com/celo-org/celo-monorepo/pulls)
+[![GitHub Issues](https://img.shields.io/github/issues-raw/celo-org/celo-monorepo.svg)](https://github.com/celo-org/celo-monorepo/issues)
+[![GitHub issues by-label](https://img.shields.io/github/issues/celo-org/celo-monorepo/1%20hour%20tasks)](https://github.com/celo-org/celo-monorepo/issues?q=is%3Aopen+is%3Aissue+label%3A%221+hour+tasks%22)
+[![GitHub issues by-label](https://img.shields.io/github/issues/celo-org/celo-monorepo/betanet-blocker)](https://github.com/celo-org/celo-monorepo/issues?q=is%3Aopen+is%3Aissue+label%3Abetanet-blocker)
```

Comment:
pls can you replace this with `betanet-phase-2` and add one for `betanet-phase-3` ?